moped-editor/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "atd-moped-editor",
   "author": "ATD Data & Technology Services",
   "license": "CC0-1.0",
-  "version": "1.15.0",
+  "version": "1.16.0",
```

Comment:
i'm not sure why this is here because both branches are supposedly up to date with main but hopefully this is okay. if not i'll uncommit this.